components/screens/full-pages/DeployToken.tsx

```diff
@@ -0,0 +1,408 @@
+import Header from "@/components/GeneralHeader.tsx";
+import { useForm } from "react-hook-form";
+import { useKasplex } from "@/hooks/useKasplex.ts";
+import { twMerge } from "tailwind-merge";
+import { Tooltip } from "react-tooltip";
+import spinner from "@/assets/images/spinner.svg";
+import React from "react";
+import useWalletManager from "@/hooks/useWalletManager.ts";
+import { setPopupPath } from "@/lib/utils.ts";
+
+type DeployFormData = {
+  ticker: string;
+  maxSupply: number;
+  mintAmount: number;
+  preAllocation: number;
+  decimalPlaces: number;
+};
+
+export default function DeployToken() {
+  const { kasplexUrl } = useKasplex();
+  const {
+    formState: {
+      errors,
+      isValid,
+      isSubmitting,
+      validatingFields,
+      touchedFields,
+    },
+    handleSubmit,
+    register,
+    watch,
+    setError,
+    clearErrors,
+  } = useForm<DeployFormData>({
+    mode: "all",
+    defaultValues: {
+      maxSupply: 100000000,
+      mintAmount: 1000,
+      preAllocation: 0,
+      decimalPlaces: 8,
+    },
+  });
+  const maxSupply = watch("maxSupply");
+  const formattedMaxSupply = Number.isNaN(maxSupply) ? 0 : maxSupply;
+  const { account } = useWalletManager();
+  const balance = account?.balance ? parseFloat(account.balance) : 0;
+  const estimatedFees = 1000.0001;
```

Comment:
```suggestion
  const estimatedFees = 1000.001;
```

I tested it, and it costs `1000.001` instead of `1000.0001`.

1000 + 995.61354751 - 995.61250891 = 0.0010386

https://explorer-tn10.kaspa.org/txs/90cb92a58327571d34869edda3b62220484e4d718b2fb4985401efdc458d7c87
https://explorer-tn10.kaspa.org/txs/088e544fb524a7fa0fab824122764631bd50b22a4b3e734ad5d2758be3b09d83